tools/mo/openvino/tools/mo/utils/ir_engine/ir_engine.py

```diff
@@ -523,16 +526,20 @@ def __read_old_api_map_order(attr, layer_type):
 
     @staticmethod
     def __read_old_api_map_element_type(attr, layer_type):
-        version = int(attr.attrib['version'])
+        version = 0
         element_type = destination_type_to_np_data_type(attr.attrib['value'])
         old_api_map = OldAPIMapElementType(version=version)
         old_api_map.set_legacy_type(element_type)
         return {('old_api_map_element_type', version): old_api_map}
 
     @staticmethod
     def __read_rt_info_common(attr):
-        attr_name = attr.attrib['name']
-        version = int(attr.attrib['version'])
+        if 'name' in attr.attrib.keys():
+            attr_name = attr.attrib['name']
+            version = int(attr.attrib['version'])
+        else:
+            attr_name = attr.name
+            version = 0
         rt_info = OrderedDict()
         for key in attr.attrib:
             if key not in ('name', 'version'):
```

Comment:
please add tests fir these changes. It can be sensitive changes

Reply:
these changes are needed to do this PR green, existed changes already will test it.

Reply:
Could you please add tests here as well: openvino\tools\mo\unit_tests\mo\utils\ir_reader\

Let me review this PR before merge. Thanks!

Reply:
We don't need to have any new tests. This PR changes the behaviour of OpenVINO serializer. All existed MO tests will fail without these changes.